src/OpenTelemetry.AutoInstrumentation/Configuration/EnvironmentConfigurationTracerHelper.cs

```diff
@@ -97,8 +97,18 @@ public static TracerProviderBuilder AddWcfInstrumentation(TracerProviderBuilder
         }
 
         [MethodImpl(MethodImplOptions.NoInlining)]
-        public static TracerProviderBuilder AddHttpClientInstrumentation(TracerProviderBuilder builder, PluginManager pluginManager)
+        public static TracerProviderBuilder AddHttpClientInstrumentation(TracerProviderBuilder builder, PluginManager pluginManager, LazyInstrumentationLoader lazyInstrumentationLoader)
         {
+            new HttpClientInitializer(lazyInstrumentationLoader, pluginManager);
+
+#if NETFRAMEWORK
+            builder.AddSource("OpenTelemetry.Instrumentation.Http.HttpWebRequest");
+#else
+            builder.AddSource("OpenTelemetry.Instrumentation.Http.HttpClient");
+            builder.AddSource("System.Net.Http"); // This works only System.Net.Http >= 7.0.0
+            builder.AddLegacySource("System.Net.Http.HttpRequestOut");
+#endif
```

Comment:
Do we need any new tests for this sources?

Reply:
We were missing .NET Framework only.
Although the `"System.Net.Http"` source was subscribed only if the target package is 7.0.0, unfortunately for us, there is no way to check it without triggering load of that package. 
In the same time, if that namespace did not exist previously, we are also good.

Reply:
@RassK could you please create an item to track any test coverage gap?

Reply:
https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/issues/1637